package.json

```diff
@@ -13,18 +16,10 @@
     "gatsby-source-filesystem": "^1.5.5",
     "gatsby-transformer-remark": "^1.7.17"
   },
-  "keywords": [
-    "gatsby"
-  ],
   "license": "MIT",
-  "main": "n/a",
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --trailing-comma es5 --no-semi --single-quote --write 'src/**/*.js'",
     "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "devDependencies": {
-    "prettier": "^1.7.4"
```

Comment:
What is the reason for removing prettier as a dev dependency, and the format script?

Reply:
If I'm understanding correctly - the only code change should be to the readme right?

Reply:
Ah apologies, I only meant to push the readme yeah!

Reply:
Thanks. I cherry-picked that commit and merged